### PR TITLE
Add coverage for create update functionality for CLI

### LINKED ISSUE #2421: Automate location update command in cli
please automate:
$ hammer location update --help
Usage:
    hammer location update [OPTIONS]

Options:
 --compute-resource-ids COMPUTE_RESOURCE_IDS Comma separated list of values.
 --compute-resources COMPUTE_RESOURCE_NAMES  Comma separated list of values.
 --config-template-ids CONFIG_TEMPLATE_IDS   Comma separated list of values.
 --config-templates CONFIG_TEMPLATE_NAMES    Comma separated list of values.
 --description DESCRIPTION                    
 --domain-ids DOMAIN_IDS                     Numerical ID or domain name
                                             Comma separated list of values.
 --domains DOMAIN_NAMES                      Comma separated list of values.
 --environment-ids ENVIRONMENT_IDS           Comma separated list of values.
 --environments ENVIRONMENT_NAMES            Comma separated list of values.
 --hostgroup-ids HOSTGROUP_IDS               Comma separated list of values.
 --hostgroups HOSTGROUP_NAMES                Comma separated list of values.
 --id ID                                     Location numeric id to search by
 --media MEDIUM_NAMES                        Comma separated list of values.
 --media-ids MEDIA_IDS                       Media IDs
                                             Comma separated list of values.
 --medium-ids MEDIUM_IDS                     Comma separated list of values.
 --name NAME                                  
 --new-name NEW_NAME                          
 --realm-ids REALM_IDS                       Numerical ID or realm name
                                             Comma separated list of values.
 --realms REALM_NAMES                        Comma separated list of values.
 --smart-proxies SMART_PROXY_NAMES           Comma separated list of values.
 --smart-proxy-ids SMART_PROXY_IDS           Comma separated list of values.
 --subnet-ids SUBNET_IDS                     Comma separated list of values.
 --subnets SUBNET_NAMES                      Comma separated list of values.
 --user-ids USER_IDS                         Comma separated list of values.
 --users USER_LOGINS                         Comma separated list of values.
 -h, --help                                  print help

### PR DESCRIPTION
**Pull request by oshtaier**
Closes #2421 

```
>nosetests tests/foreman/cli/test_location.py -m test_update_location
......................
----------------------------------------------------------------------
Ran 22 tests in 463.378s

OK
```

Continue to work in a way to add more scenarios that really make sense rather than create redundant test cases (like tests per each parameter)

### CODE REVIEW

**Comment by Ichimonji10** [comment]
ACK

File: tests/foreman/cli/test_location.py [hunk 1]
```diff
@@ -377,3 +378,214 @@ def test_create_location_with_user_by_name_negative(self):
...
+
+        result = Location.info({'id': loc['id']})
+        self.assertEqual(result.return_code, 0)
+        self.assertIn(subnet[1]['name'], result.stdout['subnets'][0])
+        self.assertIn(subnet[1]['network'], result.stdout['subnets'][0])
+
+    def test_update_location_with_multiple_comp_resources_to_single(self):
+        """@Test: Create location with multiple (not less than three) compute
+        resources assigned to it. Try to update location and overwrite all
+        compute resources by new single compute resource. Use compute resource
```
Comment (JacobCallahan) [inline tests/foreman/cli/test_location.py]: "resources with a new single" might be better

**Comment by JacobCallahan** [comment]
ACK, pending minor nitpick

**Comment by rplevka** [comment]
ACK
Comment addressed, merging..

**Pull request merged**